railgun/internal/ctrlutils/ingress-status.go

```diff
@@ -97,16 +99,20 @@ func UpdateIngress(ctx context.Context, targetContent *file.Content, log logr.Lo
 
 		switch proto := *svc.Protocol; proto {
 		case "tcp":
-			err := UpdateTCPIngress(ctx, log, svc, kiccli, ips)
-			return fmt.Errorf("failed to update tcp ingress. err %v", err)
+			if err := UpdateTCPIngress(ctx, log, svc, kiccli, ips); err != nil {
+				return fmt.Errorf("failed to update tcp ingress. err %v", err)
+			}
 		case "udp":
-			err := UpdateUDPIngress(ctx, log, svc, kiccli, ips)
-			return fmt.Errorf("failed to update udp ingress. err %v", err)
+			if err := UpdateUDPIngress(ctx, log, svc, kiccli, ips); err != nil {
+				return fmt.Errorf("failed to update udp ingress. err %v", err)
```

Comment:
```suggestion
				return fmt.Errorf("failed to update udp ingress: %w", err)
```

Reply:
Kindly see below comments and leave as it is atm.